plenum/test/replica/test_empty_lst_before_vc_if_no_prepare.py

```diff
@@ -0,0 +1,31 @@
+import pytest
+
```

Comment:
It may be sufficient to have just unit tests here:
See `test/replica/test_api.py`